grub-core/kern/efi/tpm.c

```diff
@@ -175,7 +175,7 @@ grub_tpm1_log_event(grub_efi_handle_t tpm_handle, unsigned char *buf,
 		    grub_size_t size, grub_uint8_t pcr,
 		    const char *description)
 {
-  Event *event;
+  TCG_PCR_EVENT *event;
```

Comment:
What does this change affect?  Both types appear to be equivalent.  If you want to drop `Event` usage, the unused structure definition might as well be dropped, too.  (And maybe have that in a separate commit.)

Reply:
Ah, I will try to drop the unused Event data structure in a separate PR. The data type of all the members of the Event and TCG_PCR_EVENT is the same, but the name is different. The following code just use the members from TCG_PCR_EVENT Type.